content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -87,11 +35,9 @@ timeframe; which also applies to `kubeadm`.
    private network is fine)
    
 {{< note >}}
-**Note:** This guide results in a Kubernetes cluster with one master and a
-number of nodes that you decide. A single master is not highly available. If
-you want to set up a multi-master cluster for high availability, you can
-follow
-[this guide instead](https://kubernetes.io/docs/setup/independent/high-availability/).
+**Note:** This guide results in a Kubernetes cluster with a single master and a
```

Comment:
Not a blocking issue, but IMO this note should be move  from `prerequisites` to `objectives`